sources/mysql/infoschema.go

```diff
@@ -242,6 +241,28 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 	return colDefs, colIds, nil
 }
 
+// sanitizeDefaultValue removes extra characters added to Default Value in information schema in MySQL.
+// Example_1:
+// Default Value: "concat('John', 'swamp', 'span')"
+// Default Value in MySQL infoschema with extra characters: "concat(_utf8mb4\\'John\\',_utf8mb4\\'swamp\\',_utf8mb4\\'span\\')"
+// Default Value after removing "_utf8mb4": "concat( \\'John\\',  \\'swamp\\',  \\'span\\')"
+// Default Value after removing all backslashes: "concat('John','swamp','span')"
+// Example_2:
+// Default Value: "This is a message \nwith a newline\rand a carriage return."
+// Default Value in MySQL infoschema with extra characters: "_utf8mb4\\'This is a \\tmessage \\\\nwith a newline\\\\\\'s and \\\\ra carriage return.\\'"
+// Default Value after removing "_utf8mb4": " \\'This is a \\tmessage \\\\nwith a newline\\\\\\'s and \\\\ra carriage return.\\'"
+// Default Value after removing all "\\\\": " \\'This is a \\tmessage \\nwith a newline\\\\'s and \\ra carriage return.\\'
+// Default Value after removing all "\\'": "'This is a 	message \\nwith a newline\\'s \\rand a carriage return.'
+func sanitizeDefaultValue(defaultValue string) string {
+	// replaces all occurrences of the "_utf8mb4" pattern within the defaultValue string with a single space character (" ")
```

Comment:
You can remove the comments in the code as they are redundant with the ones added on top of the function